decoder/rubix.go

```diff
@@ -546,9 +529,9 @@ func GetRubixPointNames() []string {
 }
 
 func CheckPayloadLengthRubix(data string) bool {
```

Comment:
can you please move this function to the core code somewhere. call it `checkLoRaRAWPayloadLength`. This section is unrelated to the rubix-data-encoding and actually relates to the LoRaRAW protocol which is core to this module.

This function will simply return `true` now